app/views/hq/sponsors/index.html.haml

```diff
@@ -0,0 +1,37 @@
+- if @sponsors.empty?
```

Comment:
Also keep in mind that after I wrote this I noticed that User#show contains two sponsor index views, which will most likely be a partial extracted from here (probably no action items), and in that case seeing "No Sponsors Found" would be ideal